src/components/MenuItemCard.tsx

```diff
@@ -0,0 +1,29 @@
+import { Text, HStack, MenuItem } from '@chakra-ui/react';
+import type { FunctionComponent } from 'react';
+
+import { Card } from './Card';
+
+export type MenuCardProps = {
+  icon: JSX.Element;
+  label: string;
+  textColor?: 'default' | 'red';
+  onClick?: () => void;
+};
+
+export const MenuItemCard: FunctionComponent<MenuCardProps> = ({
+  icon,
+  label,
+  textColor = 'default',
+  onClick,
+}) => {
+  return (
+    <MenuItem m="0.5rem" borderRadius="0.5rem" w="95%" onClick={onClick}>
```

Comment:
We should use standardised theme values for margins, border radii, etc.

Reply:
have moved that margin and border radius value to theme. currently we dont have global value for margin value, this rem value was getting from https://devdevout.com/px-to-rem, which 8px has been defined in our figma for margin and border radius, and rem value is 0.5rem.